app/models/spree/shipping_method.rb

```diff
@@ -3,7 +3,10 @@
 module Spree
   class ShippingMethod < ApplicationRecord
```

Comment:
Rubocop demands some refactoring. I agree that it's a good idea.

Reply:
After removing validations in https://github.com/openfoodfoundation/openfoodnetwork/pull/9262/commits/0fa25c9d860d3c811dbac9f41ea110303efdbfc2 this one is no longer an issue